examples/anchors.yaml

```diff
@@ -12,6 +12,8 @@ exporters:
       api-key: !!str 1234
     compression: gzip
     timeout: 10000
+    retry:
+      disabled: false
```

Comment:
`enabled` seems to be easier - avoids double negation

Reply:
The spec defines that options like this have to be `disabled` and not `enabled`. But I can't find it right now... so I may be wrong.

Reply:
Oh, just saw Jack already linked to it earlier, https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#boolean-value